apify-api/openapi/components/schemas/actors/UpdateActorRequest.yaml

```diff
@@ -11,6 +11,13 @@ properties:
   isPublic:
     type: boolean
     example: false
+  actorPermissionLevel:
+    type: string
+    nullable: true
```

Comment:
Can we add a `description` field with a simple text description (to both locations)? A few words and a link to docs would suffice really :)